AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -591,7 +636,8 @@ private void setupScreenLayout() {
         callHangupCancelButton.setOnClickListener(l -> closeHangupDialog());
 
         infoHeaderView = findViewById(R.id.info_header);
-        recordingActivityHeader = findViewById(R.id.recordingActivityHeader);
+        floatingHeaderArea = findViewById(R.id.recordingActivityHeader);
+        notificationActivityText = findViewById(R.id.recordingNotification);
```

Comment:
Let's rename the ids in the xml:

- recordingActivityHeader->noticeBanner
- recordingNotification->noticeBannerText
- closeRecordingNotification->closeNoticeBannerButton

Reply:
oh, I missed.. now address and updated them.